README.md

```diff
@@ -14,19 +14,14 @@ This sample uses OSIsoft Message Format to send data to OSIsoft Cloud Services,
 
 ```json
   "dependencies": {
-    "request": "2.88.2",
-    "request-promise": "4.2.5"
+    "axios": "^0.21.1"
   },
   "devDependencies": {
     "jasmine-reporters": "2.3.2",
     "jasmine": "3.5.0"
   }
 ```
 
-## Known Security Issues
```

Comment:
https://github.com/osisoft/sample-omf-complex_dynamic-nodejs/security/dependabot/package-lock.json/lodash/open is there a different issue here with 4.17.19 that we should still be announcing @dcbroad3

Reply:
I think that doesn't apply with this update. BlackDuck no longer seems to recognize this branch, but I think lodash was a dependency from request-promise that should no longer be relevant.